tutorials/scripting/gdscript/gdscript_advanced.rst

```diff
@@ -33,6 +32,8 @@ are that:
 While the main disadvantages are:
 
 -  Less performance than statically typed languages.
+-  Prone to type errors which can cause bugs that are hard
+   to trace and fix.
```

Comment:
```suggestion
-  Prone to type errors which can cause bugs that are hard to trace and fix.
```
Keep to one line, easier to read and no wider than other lines here

Reply:
done